config/file/parse_test.go

```diff
@@ -76,6 +77,8 @@ func TestParse(t *testing.T) {
 	t.Run("happy path for all extensions", func(t *testing.T) {
 		for _, ext := range supportedExt {
 			expCfg := newExpConfig()
+			fmt.Println("EXPCONFIG")
+			fmt.Println(expCfg.Request.Body)
```

Comment:
oops

Reply:
Ohlala, I did the same at Btf today... I should add a "look for all "print()" and remove them before pushing" post-it on my computer -_-